lib/popcorn/config.ex

```diff
@@ -10,6 +10,7 @@ defmodule Popcorn.Config do
         Path.wildcard("#{Application.app_dir(:iex)}/ebin/**/*.beam"),
     out_dir: nil,
     add_tracing: false,
+    runtime: {:url, "https://popcorn.swmansion.com/simple_repl/wasm/", target: :wasm},
     runtime_source: {:git, "git@github.com:software-mansion-labs/FissionVM.git"}
```

Comment:
Why we have  both `runtime` and `runtime_source`?

Reply:
I kinda don't like having two keys – if two are specified we need to either error or prioritize one over other.
What do you think about having `runtime: {:git, "<url>", type: :source | :prebuilt}`? This way we have one option and we always have opts which simplifies the code.

Reply:
Can do, but in a separate PR :P